scripts/build-iso

```diff
@@ -7,9 +7,9 @@ cd $(dirname $0)/..
 
 echo "Start building ISO image"
 
-HARVESTER_INSTALLER_VERSION=master
+HARVESTER_INSTALLER_VERSION=add-cpu-pinning-conf
```

Comment:
Please help remove the testing code. We can give the reviewers a hint of how to build a correct ISO in the PR description.